### ISSUE TITLE
Replace Travis CI with GitHub Actions?

### ISSUE THREAD

**Issue by DamienIrving** [merely-useful/py-rse#620]
In future updates to the book we may wish to replace our use of Travis CI for continuous integration with [GitHub Actions](https://github.com/features/actions).

**Comment by lwjohnst86** [comment]
Yes, 100% agree! Actions are so much easier to use and do things with than Travis, from my experience. There are a lot of features over Travis.
